### PR TITLE
Add AllowPolyBase and PolyBaseSettings properties to SqlDWSink

### PR DESCRIPTION
**Pull request by bowen5**
Add AllowPolyBase and PolyBaseSettings properties to SqlDWSink.

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@iRainbow5__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (bowan). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by markcowl** [comment]
@azuresdkci add to whitelist

**Comment by markcowl** [comment]
@akromm Can you suggest an appropriate person on the SQL team to review?

**Comment by markcowl** [comment]
@iRainbow5 We need an official review from the SQL team for this PR.  I have tagged Adam to take a look and assign a reviewer.

File: src/ResourceManagement/DataFactory/DataFactoryManagement/DataFactoryManagementChangelog.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
 ﻿For additional details on features, see the full [Azure Data Factory Release Notes](https://azure.microsoft.com/en-us/documentation/articles/data-factory-release-notes). 
 
+## Version 4.7.0
```
Comment (bgold09) [inline src/ResourceManagement/DataFactory/DataFactoryManagement/DataFactoryManagementChangelog.md]: This version number needs to be changed in AssemblyInfo.cs, the nuget.proj and .nuspec files. 

In addition, please squash these 5 commits into one. Here's an article I've found handy in the past: [Git Interactive Rebase, Squash, Amend and Other Ways of Rewriting History](https://robots.thoughtbot.com/git-interactive-rebase-squash-amend-rewriting-history)

File: src/ResourceManagement/DataFactory/DataFactoryManagement/DataFactoryManagementChangelog.md [hunk 2]
```diff
@@ -1,5 +1,13 @@
...
+_Release date: 2016.04.21_
```
Comment (bgold09) [inline src/ResourceManagement/DataFactory/DataFactoryManagement/DataFactoryManagementChangelog.md]: This needs to be updated as well.

**Comment by bgold09** [comment]
@markcowl I typically own the ADF SDK process. I've left a few comments on this, though we also do codeflow reviews internally before sending PRs.

**Comment by bgold09** [comment]
@iRainbow5 this has merge conflicts. Please resolve and also squash your commits: [Git Interactive Rebase, Squash, Amend and Other Ways of Rewriting History](https://robots.thoughtbot.com/git-interactive-rebase-squash-amend-rewriting-history)

**Comment by iRainbow5** [comment]
@bgold09 Comments resolved. Please help to take a look. Thanks.

**Comment by hovsepm** [comment]
@azuresdkci add to whitelist

**Comment by hovsepm** [comment]
@azuresdkci retest this please

**Comment by bgold09** [comment]
@iRainbow5 @markcowl Looks good, please merge.

**Pull request merged**